.travis.yml

```diff
@@ -19,4 +19,5 @@ install:
 
 script:
     - ./run-checks --static || travis_terminate 1
+    - ./run-checks --unit || travis_terminate 1
```

Comment:
I think thats a good idea. The downside of the unittest as an extra step is that it increases the test time a little bit. But the upside is that failing unittests will be noticed right away.